daemon/openrazer_daemon/dbus_services/dbus_methods/mamba.py

```diff
@@ -348,3 +348,54 @@ def get_supported_poll_rates(self):
     self.logger.debug("DBus call get_supported_poll_rates")
 
     return self.POLL_RATES
+
+
+@endpoint('razer.device.misc', 'setHyperPollingLED', in_sig='y')
+def set_hyperpolling_wireless_dongle_indicator_led_mode(self, mode):
+    """
+    Set the LED mode, takes in 1 char
+
+    :param mode: LED mode
+    :type mode: char
+    """
+    self.logger.debug("DBus call set_HyperPollingLED")
+
+    driver_path = self.get_driver_path('hyperpolling_wireless_dongle_indicator_led_mode')
+
+    # remember LED mode
+    self.ledmode = mode
```

Comment:
Is this used anywhere? I don't see anything

Reply:
Same as previous comment, this is to expose the LED functionality which the user can set via sysfs or that a frontend can implement. Or if you're referring specifically to the self.ledmode, it's currently unused. I believe I just copied the format/structure/style from a similar function elsewhere in the file and wasn't sure if remembering the status was 'best practice'.

Reply:
Yeah let's remove if it's not used.

Reply:
Done!